.gitignore

```diff
@@ -14,3 +14,4 @@ karma-*
 yarn-error.log
 coverage
 webpack.report.html
+*.sublime-*
```

Comment:
I think this kind of gitignore belongs in each developer's global gitignore. That way you add it to your global gitignore once in your life and you don't have to add it to the gitignore of every project you work with.